lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -52,6 +53,10 @@ object NotificationHandler extends CohortHandler {
     }
   }
 
+  def currencyISOtoSymbol(iso: String): ZIO[Any, Nothing, String] = {
+    ZIO.succeed(i18n.Currency.fromString(iso: String).map(_.identifier).getOrElse(""))
+  }
```

Comment:
If this can't fail, it doesn't need to be wrapped in a `ZIO`:
```
private def currencyISOtoSymbol(iso: String): String = 
    i18n.Currency.fromString(iso: String).map(_.identifier).getOrElse("")
```

but arguably it should fail or put the sub into a different state if there's no corresponding symbol (eg if some new currency emerges.)